src/lib/adamant-api/index.js

```diff
@@ -129,6 +129,74 @@ export function getPublicKey(address = '') {
   })
 }
 
+
+/**
+ * @typedef {Object} MessageObject
+ * @property {number} amount
+ * @property {Object} asset
+ * @property {Object} asset.chat
+ * @property {string} asset.chat.message
+ * @property {string} asset.chat.own_message
+ * @property {number} asset.chat.type
+ * @property {string} recipientId
+ * @property {string} senderId
+ * @property {string} senderPublicKey
+ * @property {string} signature
+ * @property {number} timestamp
+ * @property {number} type
+ */
```

Comment:
Typing here is redundant. We already have types in `index.d.ts`:

https://github.com/Adamant-im/adamant-im/blob/34afe48aaf79588ee10c36d61d6c8bf5a19a9257/src/lib/adamant-api/index.d.ts#L93

Just use `Promise<object>`. This is enough. We will merge `index.js` and `index.d.ts` in the future.